cura/PrinterOutput/NetworkedPrinterOutputDevice.py

```diff
@@ -178,7 +178,7 @@ def put(self, target: str, data: str, onFinished: Optional[Callable[[Any, QNetwo
         if onFinished is not None:
             self._onFinishedCallbacks[reply.url().toString() + str(reply.operation())] = onFinished
 
-    def get(self, target: str, onFinished: Optional[Callable[[Any, QNetworkReply], None]]):
+    def get(self, target: str, onFinished: Optional[Callable[[QNetworkReply], None]]) -> None:
```

Comment:
This change is incorrect; the _onFinished functions need to be a callable with an any type as first param and a QNetwork reply as second.